Lib/test/test_msvcrt.py

```diff
@@ -11,8 +13,6 @@
 import _winapi
 import msvcrt;
```

Comment:
It's not C or C++ :-)

```suggestion
import msvcrt
```

You can also move ``if sys.platform != "win32":`` test before "from test..." imports.

Reply:
The imports behind the  `if sys.platform != ...` test are modules that can't be used on non Windows platforms. The `test.support` can be used in these non Windows platforms, so I think we should group them like current codes?